lib/danger/request_source/github.rb

```diff
@@ -231,6 +231,10 @@ def process_markdown(violation)
         message = match.nil? ? html : match.captures.first
         Violation.new(message, violation.sticky)
       end
+
+      def organisation
```

Comment:
Probably worth putting in comments here about how it's used inside danger, otherwise it feels arbitrary